tools/odbc/include/api_info.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef API_INFO_HPP
+#define API_INFO_HPP
+
+#include "duckdb.hpp"
+
+#include <sql.h>
+#include <sqltypes.h>
+#include <sqlext.h>
+#include <map>
+#include <vector>
+
+#define NUM_FUNC_SUPPORTED 4000
+
+namespace duckdb {
+
+struct ApiInfo {
+
+private:
+	// fill all supported functions in this array
+	static const std::vector<SQLUSMALLINT> ALL_SUPPORTED_FUNCTIONS;
+
+	// fill ODBC3 supported functions in this array
+	static const std::vector<SQLUSMALLINT> ODBC3_SUPPORTED_FUNCTIONS;
+
+	static const std::vector<SQLSMALLINT> ODBC_SUPPORTED_SQL_TYPES;
+
+	struct LogicalTypeCmp {
+		bool operator()(const LogicalTypeId a, const LogicalTypeId b) const {
+			return a < b;
+		}
+	};
+	// Map DuckDB LogicalTypes to ODBC SQL Types
+	static const std::map<duckdb::LogicalTypeId, SQLSMALLINT, LogicalTypeCmp> MAP_SQL_TYPES;
```

Comment:
Could this map be moved into the `FindRelatedSQLType` function and turned into a `switch` statement?

Reply:
Done. CI is in progress now. ;)